library/src/visualize/image.rs

```diff
@@ -22,7 +22,7 @@ use crate::prelude::*;
 ///
 /// Display: Image
 /// Category: visualize
-#[element(Layout)]
+#[element(Locatable, Layout)]
```

Comment:
not required anymore, right?

Reply:
I figure some people may still want to locate images?

Reply:
I mean, that might want to locate anything. For now, I would add `Locatable` only when required by Typst and I will try to fix the need for this hack once and for all later.